db/seeds.rb

```diff
@@ -1,7 +1,56 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+Game.destroy_all
```

Comment:
I think we can turn this check into something like `return unless Game.count == 0`. If we run `db:reset` or something like that, it'll destroy the records anyway.

Reply:
Yeah, that makes way more sense.